src/js/common/schemaform/review/ObjectField.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import _ from 'lodash/fp';
+import classNames from 'classnames';
```

Comment:
Are the changes in here still necessary?

Reply:
Nice catch, removed.